resources/wikia/modules/geo.js

```diff
@@ -42,7 +42,8 @@
 		 */
 		function getCountryCode() {
 			var data = getGeoData();
-			return data.country;
+			//return data.country;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ACommentedCode)